Sources/ComposableArchitecture/Dependencies/Dismiss.swift

```diff
@@ -60,17 +60,17 @@ extension DependencyValues {
 /// > some shared mutable state inside the `dismiss` closure to confirm that it is indeed invoked:
 /// >
 /// > ```swift
-/// > let isDismissInvoked = LockIsolated(false)
-/// > let store = Store(initialState: Child.State()) {
+/// > let isDismissInvoked: LockIsolated<[Bool]> = .init([])
```

Comment:
I've replaced `LockIsolated<Bool>` with its array-based counterpart, `LockIsolated<[Bool]>`. This isn't a correction of a typo, but rather a small improvement for reliability. It ensures that dismiss is not called more than once.